src/registrar/models/user_portfolio_permission.py

```diff
@@ -15,8 +15,6 @@ class Meta:
     PORTFOLIO_ROLE_PERMISSIONS = {
         UserPortfolioRoleChoices.ORGANIZATION_ADMIN: [
             UserPortfolioPermissionChoices.VIEW_ALL_DOMAINS,
-            UserPortfolioPermissionChoices.VIEW_MEMBERS,
-            UserPortfolioPermissionChoices.EDIT_MEMBERS,
```

Comment:
I discussed this with Alysia.  It is regarding the AC where admins cannot, by default, view or edit members.  The AC says "this needs to be layered on", and it was clarified to me that this meant deleting these permissions from admin so that they have to be explicitly added in "additional permissions"